lib/vauth/ntlm.c

```diff
@@ -451,6 +451,7 @@ CURLcode Curl_auth_create_ntlm_type1_message(struct Curl_easy *data,
             LONGQUARTET(NTLMFLAG_NEGOTIATE_OEM |
                         NTLMFLAG_REQUEST_TARGET |
                         NTLMFLAG_NEGOTIATE_NTLM_KEY |
+                        NTLMFLAG_NEGOTIATE_NTLM2_KEY |
                         NTLM2FLAG |
```

Comment:
``NTLM2FLAG`` already contains ``NTLMFLAG_NEGOTIATE_NTLM2_KEY``, see line ~408:
```
#if defined(USE_NTRESPONSES) && defined(USE_NTLM2SESSION)
#define NTLM2FLAG NTLMFLAG_NEGOTIATE_NTLM2_KEY
#else
#define NTLM2FLAG 0
#endif
```

Reply:
Oh I overlooked that. Thank you